ui/packages/catalog-ui-search/src/main/webapp/component/confirmation/query/confirmation.query.less

```diff
@@ -1,18 +1,29 @@
 @{customElementNamespace}confirmation.is-query {
   .confirmation-query {
     padding: @largeSpacing;
+    flex-grow: 1;
+  }
+
+  .search-replace-wrapper {
+    display: flex;
+    align-items: center;
   }
 
   .confirmation-replace {
     display: none;
   }
 
   .confirmation-choices {
+    display: flex;
     button {
-      width: 50%;
+      width: auto;
+      border-radius: 5px;
```

Comment:
Buttons it Intrigue shouldn't have rounded corners.

Reply:
See above comment. I don't disagree, I was working off of a mockup that was provided to me.

Reply:
Generally we should be using mockups should be a guide for general layout and functionality. Styles should be consistent with what we do in the UI already.